src/crypto/tls/common.go

```diff
@@ -1296,6 +1316,8 @@ type Certificate struct {
 	// SignedCertificateTimestamps contains an optional list of Signed
 	// Certificate Timestamps which will be served to clients that request it.
 	SignedCertificateTimestamps [][]byte
+	// DelegatedCredential is serialized Delegated Credential
```

Comment:
To a developer this comment is probably not very helpful. Perhaps something like:
```suggestion
	// DelegatedCredential is a serialized Delegated Credential signed by
	// the leaf certificate. See
	// https://tools.ietf.org/html/draft-ietf-tls-subcerts-09#section-4
```

Reply:
Yes! Thank you.